rrtmgp/mo_gas_optics_rrtmgp.F90

```diff
@@ -489,13 +489,13 @@ function compute_gas_taus(this,                       &
     ! Compute dry air column amounts (number of molecule per cm^2) if user hasn't provided them
     !
     idx_h2o = string_loc_in_array('h2o', this%gas_names)
-    !$acc enter data create(col_dry_wk, col_dry_arr, col_gas)
     if (present(col_dry)) then
       col_dry_wk => col_dry
     else
       col_dry_arr = get_col_dry(vmr(:,:,idx_h2o), plev) ! dry air column amounts computation
       col_dry_wk => col_dry_arr
     end if
+    !$acc enter data create(col_dry_wk, col_dry_arr, col_gas)
```

Comment:
@inpolonsky Do you understand why this !$acc enter data create statements is after the pointer assignment in your branch and above the pointer assignment on develop? This seems like a place where the code should not be changing.